tests/integration/base.yaml

```
@@ -12,10 +12,11 @@ spec:
       proxyMetadata:
         ISTIO_META_DNS_CAPTURE: "true"
   values:
+    global:
+      variant: "" # we decide variant in framework
```

```suggestion
    global:
      variant: "" # we decide variant in framework
```

Move this down to the other global section, otherwise, lgtm